db/schema.rb

```diff
@@ -1376,7 +1376,6 @@
     t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true
     t.index ["twitter_username"], name: "index_users_on_twitter_username", unique: true
     t.index ["username"], name: "index_users_on_username", unique: true
-    t.check_constraint "username IS NOT NULL", name: "users_username_not_null"
```

Comment:
@msarit is this a related schema change?

Reply:
Nope, it's not 😟

Reply:
this should be re-added in place